### DISCUSSION TITLE
[Feature] Add weather to main screen

### DISCUSSION THREAD

**Discussion by dhiorns15** [damongolding/immich-kiosk#129] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would be interested in adding weather to the main screen similar to what [Google Hub](https://cdn.vox-cdn.com/thumbor/hFLM2g3I9qVtJ5rLJzcYBd-uNpE=/0x0:2040x1360/1400x1400/filters:focal(1020x680:1021x681)/cdn.vox-cdn.com/uploads/chorus_asset/file/13299557/dseifert_181018_3039_1325.jpg) has. [OpenWeatherMap](https://openweathermap.org/api) is a great tool that I have seen in the past implemented into projects like Magic Mirror.

**Comment by damongolding** [comment]
There's a couple of things that would need to be considered with this. 

I assume the API would sit in config/env as would the location but seeing as Kiosk supports multiple clients should/could those override any of those settings?

**Comment by dhiorns15** [comment]
You probably can specify multiple locations in the config and pass a variable in the URL for which location to use. Just spitballin here.

**Comment by damongolding** [comment]
Spitballing is completely welcome and encouraged!

That could work like so maybe?

```yaml
weather: 
  - name: london
    api: ""
    lat: ""
    long: ""

  - name: new_york
    api: ""
    etc ...
```

**Comment by damongolding** [comment]
Implemented in v0.12.0
